effectivity/README.md

```diff
@@ -0,0 +1,159 @@
+--- 
+layout: pattern
+title: Effectivity
+folder: effectivity
+permalink: /patterns/effectivity/
+categories: Creational
+language: en 
+tags:
+ - Data access
+---
+
+## Name / classification
+
+Effectivity.
+
+## Also known as
+
+Effective Dating, Temporal Database Design.
+
+## Intent
+
+Add a time period to an object to show when it is effective.
+
+## Explanation
+
+Real-world example
+
+> An individual is employed at different companies over different periods of time during their 
```

Comment:
Probably reword " a certain position is effective" into more commonly used language for a real world example.

Reply:
Thanks Ethan, it is quite confusing. I've changed that part to 'It is useful to know if they were working at a company at certain times.'